src/main/scala/fr/acinq/bitcoin/BtcAmount.scala

```diff
@@ -1,6 +1,11 @@
 package fr.acinq.bitcoin
 
-sealed trait BtcAmount
+sealed trait BtcAmount {
+  def toMilliSatoshi: MilliSatoshi
+  def toSatoshi: Satoshi = millisatoshi2satoshi(toMilliSatoshi)
+  def toMilliBtc: MilliBtc = satoshi2millibtc(toSatoshi)
+  def toBtc: Btc = millibtc2btc(toMilliBtc)
+}
 
 case class Satoshi(amount: Long) extends BtcAmount {
```

Comment:
It's a bit unrelated but I think the `amount` field should be made private here, and has to be accessed with `toLong`. It's what we did for the `UInt64` class and I find it cleaner.

```suggestion
case class Satoshi(private val amount: Long) extends BtcAmount {
```